Mathlib/Algebra/Category/GroupCat/Kernels.lean

```diff
@@ -0,0 +1,70 @@
+/-
+Copyright (c) 2023 Moritz Firsching. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Kurniadi Angdinata, Moritz Firsching, Nikolas Kuhn
+-/
+import Mathlib.Algebra.Category.GroupCat.EpiMono
+import Mathlib.CategoryTheory.Limits.Shapes.Kernels
+
+/-!
+# The concrete (co)kernels in the category of abelian groups are categorical (co)kernels.
+-/
+
+namespace AddCommGroupCat
+
+open CategoryTheory Limits
+
+universe u
+
+variable {G H : AddCommGroupCat.{u}} (f : G ⟶ H)
+
+-- TODO: relocate theorem
+theorem range_le_ker_iff {G H I : AddCommGroupCat.{u}} {f : G ⟶ H} {g : H ⟶ I} :
+    f.range ≤ g.ker ↔ g.comp f = 0 :=
+  ⟨fun h => AddMonoidHom.ext fun x => h ⟨x, rfl⟩,
+    by rintro h _ ⟨x', rfl⟩; exact FunLike.congr_fun h x'⟩
+
+-- TODO: relocate theorem
```

Comment:
Is this a to-do for now or for later? And the other ones nearby

Reply:
Whoops I completely forgot about these - I'll relocate them now

Reply:
Whoops I completely forgot about these - I've relocated them.